ResetPasswordBundle/src/PasswordResetHelper.php

```diff
@@ -76,67 +109,67 @@ public function generateResetToken(object $user): PasswordResetToken
         );
     }
 
-    public function validateTokenAndFetchUser(string $fullToken): object
+    /**
+     * Validate a PasswordResetRequest and fetch user from database
+     *
+     * @param string $fullToken selector + non-hashed verifier token
+     * @return UserInterface
+     * @throws ExpiredResetPasswordTokenException
+     * @throws InvalidResetPasswordTokenException
+     * @throws \Throwable @TODO Refactor TokenGenerator and change exception signature
```

Comment:
I would remove all of these except for the first `@param` and the first 2 `@throws`. We try to only keep phpdoc that *adds* more information - I think those both do.

Reply:
done